### DISCUSSION TITLE
How to sync added files in one go?

### DISCUSSION THREAD

**Discussion by ShubhamVerma1811** [twpayne/chezmoi#3190] [Q&A] [answered]
Hey,
Thanks for building chezmoi.

I wanted to know is there a way to auto add/update/edit all the added files with a command?

For example:
`chezmoi add ~/.bashrc`  creates a  dot_bashrc.

Now, for the next time instead of again running the add command, is there a way to auto sync the file?

Sync chezmoi already knows the path, can it pull the local file and update it the chezmo's cd?

**Comment by twpayne** [comment, marked as answer]
https://www.chezmoi.io/reference/commands/re-add/ ?

**Reply by Lockszmith-GH** [reply]
This is most likely the answer, if not, and you're looking for a deeper discussion, take a look at #2852 which might provide better understanding.

**Reply by ShubhamVerma1811** [reply]
Yup, re-add it is!

Thank you!
